app/views/courses/_advice.html.erb

```diff
@@ -1,15 +1,12 @@
 <div class="govuk-!-margin-bottom-8">
   <h2 class="govuk-heading-l" id="section-advice">Support and advice</h2>
-  <p class="govuk-body">For support and advice, you can speak to a <%= govuk_link_to 'teacher training adviser', t('get_into_teaching.url_get_an_advisor') %>
-  for free. They’re all experienced teachers who can help you to prepare your application, book school
-    experience, and access exclusive teaching events.</p>
-  <p class="govuk-body">You can also call <%= t('service_name.get_into_teaching') %> free of charge on <%= t('get_into_teaching.tel') %>, or
+  <p class="govuk-body">You can speak to a <%= govuk_link_to 'teacher training adviser', t('get_into_teaching.url_teacher_training_advisor') %>
+  for free, dedicated advice. They are all experienced teachers who can give you one-to-one support with your application. They can also help you
+    get school experience, and find events where you can speak to training providers and teachers.</p>
+  <p class="govuk-body">Call Get Into Teaching free of charge on <%= t("get_into_teaching.tel") %>, or
     <%= govuk_link_to 'chat online', t('get_into_teaching.url_online_chat') %>
     <%= t('get_into_teaching.opening_times') %> (except public holidays).</p>
 
-  <p class="govuk-body">Find out about the different
-  <%= govuk_link_to('ways to train', t('get_into_teaching.url_ways_to_train')) %>
-  if you do not have a degree, or are already an experienced teacher.</p>
-
+  <h3 class="govuk-heading-m">Is there something wrong with this page?</h3>
```

Comment:
Not part of this card, but i've updated this section in line with the prototype as I noticed they were different @Kateharries @simonwhatley is that ok?

Reply:
Hello. Yes, that's fine - thanks @avinhurry :)